### PR TITLE
[EN-1228] ci cd

### PR DESCRIPTION
**Pull request by iamishalkin**
One workflow action that tests package installation, documentation and testing. Triggered on push in master branch

### CODE REVIEW

File: .github/workflow/pythonpackage.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      run: |
+        poetry config virtualenvs.create false
+        poetry install --no-root --no-dev
+        pip install taskipy cython
+        task build -f sdist
+        pip uninstall --yes taskipy cython
+      shell: bash
+    - name: Install package artifact
+      run: |
+        pip install dist/cyrtd*
```
Comment (alimantu) [inline .github/workflow/pythonpackage.yml:43]: Seems to be misstype here.

File: .github/workflow/pythonpackage.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: true
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install poetry
```
Comment (alimantu) [inline .github/workflow/pythonpackage.yml:29]: Why we are not using github actions for this? -> https://github.com/marketplace/actions/setup-poetry

File: .github/workflow/pythonpackage.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      shell: bash
+    - name: Run tests
+      run: |
+        task test
+      shell: bash
+    - name: Generate doc
+      run: |
+        task html_docs
+      shell: bash
+      if: matrix.os == 'ubuntu-latest'
```
Comment (alimantu) [inline .github/workflow/pythonpackage.yml:80]: Do we really want to generate documentation 3 times in a row? For each python version specified on top 🤔

File: .github/workflow/pythonpackage.yml (hunk 2 above)
Comment (iamishalkin) [reply .github/workflow/pythonpackage.yml:29]: I had issues with two actions for poetry installation for different os. Moreover, I suppose, this action is not official one, so pip installation looks more reliable

Comment (alimantu) [reply .github/workflow/pythonpackage.yml:29]: Ok, then we'll use this workaround till release of the official action or at least semi-official one.

File: .github/workflow/pythonpackage.yml (hunk 3 above)
Comment (iamishalkin) [reply .github/workflow/pythonpackage.yml]: corrected

File: .github/workflow/pythonpackage.yml (hunk 1 above)
Comment (iamishalkin) [reply .github/workflow/pythonpackage.yml]: corrected

**Pull request merged**